frontend/src/scenes/events/EventDetails.js

```diff
@@ -2,25 +2,42 @@ import React from 'react'
 import { PropertiesTable } from 'lib/components/PropertiesTable'
 import moment from 'moment'
 import { EventElements } from 'scenes/events/EventElements'
-import { Tabs } from 'antd'
+import { Tabs, Button } from 'antd'
+
+import { createActionFromEvent, recurseSelector } from './createActionFromEvent'
 const { TabPane } = Tabs
 
 export function EventDetails({ event }) {
     return (
-        <Tabs data-attr="event-details" defaultActiveKey="properties" animated={false}>
-            <TabPane tab="Properties" key="properties">
-                <PropertiesTable
-                    properties={{
-                        Timestamp: moment(event.timestamp).toISOString(),
-                        ...event.properties,
-                    }}
-                />
-            </TabPane>
-            {event.elements.length > 0 && (
-                <TabPane tab="Elements" key="elements">
-                    <EventElements event={event} />
+        <>
+            {event.elements.length > 0 && recurseSelector(event.elements, '', 0)}
```

Comment:
Not quite sure how you meant to display the selector. Right now it's colliding:
<img width="617" alt="Screen Shot 2020-06-08 at 12 23 50 PM" src="https://user-images.githubusercontent.com/13127476/84055581-094be480-a983-11ea-9519-7627ee77205a.png">

Reply:
Oops, that was mean to be for debugging. Removed